### PR TITLE
Interior plugin added

### PR DESCRIPTION
**Pull request by jonasesser**

### CODE REVIEW

**Comment by Stuyk** [comment]
Needs to be updated @jonasesser thanks. Check the original templates again.

File: plugins/j0n4s/interior-plugin.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+
+layout: ../../../layouts/Main.astro
```
Comment (Stuyk) [inline plugins/j0n4s/interior-plugin.md]: Layout path needs update.

File: plugins/j0n4s/interior-plugin.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+
+layout: ../../../layouts/Main.astro
+title: "Interior Plugin"
+description: "This Plugin helps you to find an interior including position and ipl informations"
+author: "j0n4s"
+version: "3.0.0+"
+images: []
```
Comment (Stuyk) [inline plugins/j0n4s/interior-plugin.md]: Needs at least one image.

**Review by Stuyk — CHANGES_REQUESTED** [review]

File: plugins/j0n4s/interior-plugin.md (hunk 1 above)
Comment (jonasesser) [reply plugins/j0n4s/interior-plugin.md]: ok

File: plugins/j0n4s/interior-plugin.md (hunk 2 above)
Comment (jonasesser) [reply plugins/j0n4s/interior-plugin.md]: added

**Pull request merged**